pdftotree/utils/pdf/pdf_parsers.py

```diff
@@ -1274,11 +1275,8 @@ def get_page_width(boxes):
 
 
 def get_char_width(boxes: List[LTTextLine]) -> float:
-    box_len_sum = 0
-    num_char_sum = 0
-    for i, b in enumerate(boxes):
-        box_len_sum = box_len_sum + b.bbox[2] - b.bbox[0]
-        num_char_sum = num_char_sum + len(b.get_text())
+    box_len_sum = np.sum([b.bbox[2] - b.bbox[0] for b in boxes] )
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**chore:** ‏remove the extra space before the final parenthesis

Reply:
This is done. Thank you